### PR TITLE
Able to enable/disable Monitoring Policy 

### PR DESCRIPTION
**Pull request by Mohammad-Junid**
## Description
Monitoring policy should have an option to enable/disable.   

## Jira link(s)
- https://opennms.atlassian.net/browse/LOK-2439

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

**Comment by aurang-zeb313** [comment]
Approved.

**Review by aurang-zeb313 — APPROVED** [review]
Approved

File: alert/src/test/resources/org/opennms/horizon/alertservice/monitor-policy.feature [hunk 1]
```diff
@@ -56,6 +57,17 @@ Feature: Monitor policy gRPC Functionality
...
 
+
+  Scenario: Create  monitor policy with disabled status
+    Given Tenant id "test-tenant"
+    Given A monitoring policy named "disabled-rule-policy" with tag "tag1" and with  status disabled
+    Given The policy has a rule named "new-rule" with component type "NODE" and trap definitions
+      | trigger_event_name | count | overtime | overtime_unit | severity | clear_event_name |
+      | SNMP Cold Start    | 1     | 3        | MINUTE        | MAJOR    |                  |
+    And The monitor policy is created in the tenant with disabled status
+    Then verify that the status on monitor status is disabled
```
Comment (cgorantla) [inline alert/src/test/resources/org/opennms/horizon/alertservice/monitor-policy.feature:68]: use name "disabled-rule-policy" to get matching policy

**Review by cgorantla — COMMENTED** [review]
Looks good. Can you validate this with GraphQL API

File: alert/src/test/resources/org/opennms/horizon/alertservice/monitor-policy.feature (hunk 1 above)
Comment (Mohammad-Junid) [reply alert/src/test/resources/org/opennms/horizon/alertservice/monitor-policy.feature]: Changes are done

**Comment by Mohammad-Junid** [comment]
image
image
image
image
image

**Review by cgorantla — APPROVED** [review]

**Comment by Mohammad-Junid** [comment]
LOK-2439  merge to develop

**Pull request merged**